ee/codegen/src/project.ts

```diff
@@ -416,12 +416,15 @@ ${errors.slice(0, 3).map((err) => {
       );
     }
 
-    const nodesToGenerate = await Promise.all(
+    const nodesToGenerate = await Promise.allSettled(
       this.getOrderedNodes().map(async (nodeData) => {
-        await createNodeContext({
-          workflowContext: this.workflowContext,
-          nodeData,
-        }).catch((error) => {
+        try {
+          await createNodeContext({
```

Comment:
create a test that reproduces this issue by mocking this `createNodeContext` to throw an error once. Should pass with this PR but fail on main